src/test/java/org/eclipse/tractusx/ssi/lib/util/TestResourceUtil.java

```diff
@@ -139,4 +140,14 @@ private static InputStream readResource(String resource) {
 
     return Objects.requireNonNull(inputStream, "Resource not found: " + resource);
   }
+
+  public static VerifiableCredentialStatusList2021Entry getStatusListEntry() {
```

Comment:
to keep the terminology clean, I'd  remove 2021, since we are using BitStringStatusList https://www.w3.org/TR/vc-bitstring-status-list/

Reply:
Fixed in https://github.com/Cofinity-X/SSI-agent-lib/pull/10/commits/4dfee1c4a73f528a8205eb3b165c976303197b1a